test/integration/targets/ansible-playbook-callbacks/all-callbacks.yml

```diff
@@ -96,26 +96,22 @@
       ignore_errors: true
 
     - name: async poll ok
-      command: sleep 2
-      async: 3
-      poll: 1
+      command: sleep 3
+      async: 5
+      poll: 2
 
     - name: async poll failed
-      shell: sleep 2; false
-      async: 3
-      poll: 1
+      shell: sleep 3; false
+      async: 5
+      poll: 2
       ignore_errors: true
 
-    - include_tasks: include_me.yml
-
     - name: diff
       copy:
         content: diff
-        dest: '{{ remote_tmp_dir.path }}/diff.txt'
+        dest: '{{ remote_tmp_dir }}/diff.txt'
       diff: true
 
-- hosts: i_dont_exist
```

Comment:
This also shouldn't be removed. Instead run the playbook with `ANSIBLE_HOST_PATTERN_MISMATCH=warning` (set temporarily just for this playbook execution) so the callback is tested.

Reply:
Will do.